File: CS-Examples/14_Hyperlinks/ModifyHyperlink.cs
using System;
using System.Windows.Forms;
using Spire.Xls;
using Spire.Xls.Collections;

namespace ModifyHyperlink
{

	public partial class Form1 : Form
	{
        public Form1()
        {
            InitializeComponent();
        }

		private void btnRun_Click(object sender, EventArgs e)
		{
            // Create a new workbook
            Workbook workbook = new Workbook();

            // Load an existing workbook from a file
            workbook.LoadFromFile(@"..\..\..\..\..\..\Data\ModifyHyperlink.xlsx");

            // Get the first worksheet in the workbook
            Worksheet sheet = workbook.Worksheets[0];

            // Get the collection of all hyperlinks in the worksheet
            HyperLinksCollection links = sheet.HyperLinks;

            // Modify the values of TextToDisplay and Address properties of the first hyperlink
            links[0].TextToDisplay = "Spire.XLS for .NET";
            links[0].Address = "http://www.e-iceblue.com/Introduce/excel-for-net-introduce.html";

            // Specify the output file name for the modified workbook
            string output = "ModifyHyperlinkResult.xlsx";

            // Save the modified workbook to the specified file using Excel 2013 format
            workbook.SaveToFile(output, ExcelVersion.Version2013);

            // Dispose of the workbook object to release resources
            workbook.Dispose();

            // Launch the Excel file
            ExcelDocViewer(output);
		}
        private void ExcelDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }

        private void btnClose_Click(object sender, EventArgs e)
        {
            Close();
        }

	}
}

File: CS-Examples/14_Hyperlinks/ModifyHyperlink.vb
Imports Spire.Xls
Imports Spire.Xls.Collections

Namespace ModifyHyperlink

	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
            ' Create a new workbook object
            Dim workbook As New Workbook()

            ' Load an existing workbook from the specified file path
            workbook.LoadFromFile("..\..\..\..\..\..\Data\ModifyHyperlink.xlsx")

            ' Get the first worksheet in the workbook
            Dim sheet As Worksheet = workbook.Worksheets(0)

            ' Get the collection of hyperlinks in the worksheet
            Dim links As HyperLinksCollection = sheet.HyperLinks

            ' Update the display text and address of the first hyperlink in the collection
            links(0).TextToDisplay = "Spire.XLS for .NET"
            links(0).Address = "http://www.e-iceblue.com/Introduce/excel-for-net-introduce.html"

            ' Define the output file name as "ModifyHyperlinkResult.xlsx"
            Dim output As String = "ModifyHyperlinkResult.xlsx"

            ' Save the modified workbook to the specified file path using Excel 2013 format
            workbook.SaveToFile(output, ExcelVersion.Version2013)

            ' Release the resources used by the workbook
            workbook.Dispose()

            'Launch the Excel file
            ExcelDocViewer(output)
		End Sub
		Private Sub ExcelDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub

		Private Sub btnClose_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnClose.Click
			Close()
		End Sub

	End Class
End Namespace
